components/Downloads/PrimaryDownloadMatrix.tsx

```diff
@@ -2,43 +2,37 @@ import classNames from 'classnames';
 import semVer from 'semver';
 import LocalizedLink from '../LocalizedLink';
 import { useNextraContext } from '../../hooks/useNextraContext';
-import type { NodeVersionData, LegacyDownloadsFrontMatter } from '../../types';
+import type { LegacyDownloadsFrontMatter, NodeReleaseData } from '../../types';
 import type { FC } from 'react';
 
-type PrimaryDownloadMatrixProps = Pick<
-  NodeVersionData,
-  'isLts' | 'node' | 'nodeNumeric' | 'npm'
->;
+type PrimaryDownloadMatrixProps = {
+  release: NodeReleaseData;
+};
 
 // @TODO: Instead of using a static list it should be created dynamically. This is done on `nodejs.dev`
 // since this is a temporary solution and going to be fixed in the future.
-const PrimaryDownloadMatrix: FC<PrimaryDownloadMatrixProps> = ({
-  node,
-  nodeNumeric,
-  npm,
-  isLts,
-}) => {
+const PrimaryDownloadMatrix: FC<PrimaryDownloadMatrixProps> = ({ release }) => {
```

Comment:
Can we also transform the prop type here to:

```suggestion
const PrimaryDownloadMatrix: FC<NodeReleaseData> = ({ npm, version, isLts }) => {
```

Reply:
👀 @HinataKah0